irohad/network/impl/block_loader_impl.cpp

```diff
@@ -23,6 +24,7 @@ namespace {
   const char *kPeerNotFound = "Cannot find peer";
   const char *kPeerRetrieveFail = "Failed to retrieve peers";
   const char *kPeerFindFail = "Failed to find requested peer";
+  const std::chrono::seconds kBlocksRequestTimeout{5};
```

Comment:
May be it would be better to make this value configurable?

Reply:
If I recall right, this question was brought up on some meeting, and we decided to hard-code it. Let's ask @lebdron.